### ISSUE TITLE
API error: Cannot read property 'trace' of undefined

### ISSUE THREAD

**Issue by bkuhl** [rollbar/rollbar-php#95]
I'm unsure if this is an issue with this package or something within Rollbar.  I just added this to my Laravel 5 project using https://github.com/jenssegers/laravel-rollbar and am testing things out and Rollbar is seeing an exception, but it's not the one I'm throwing.

I've verified that the exception and appropriate context is being passed to the rollbar-php package.

<img width="961" alt="screen shot 2016-06-11 at 1 59 08 pm" src="https://cloud.githubusercontent.com/assets/524933/15986702/b4eb1d80-2fdc-11e6-98f7-54de22cbb6ec.png">
<img width="712" alt="screen shot 2016-06-11 at 2 01 52 pm" src="https://cloud.githubusercontent.com/assets/524933/15986714/1685704a-2fdd-11e6-9ef9-a0ce6bdcca01.png">

@dlsteuer began investigating this issue with me and we've narrowed it down to https://github.com/rollbar/rollbar-php/blob/master/src/rollbar.php#L342-L343 being the cause of the issue.  I've created this ticket for tracking purposes.

**Comment by bkuhl** [comment]
My app needs to go in production in the next month and this is a blocker for me.  I'm happy to Fork the project and disable the conversion that appears to be breaking this, but would prefer to only take that route if this isn't something that is on the radar for a fix.

**Comment by dlsteuer** [comment]
Hey @bkuhl we're working towards the 1.0 release, which wouldn't have this problem because it's been removed in that branch, I need to have a discussion with some other members of the team about that. The only thing holding back the 1.0 release right now is api compatibility with the current version.

**Comment by bkuhl** [comment]
Any idea on what the anticipated date of that is?

**Comment by dlsteuer** [comment]
Hey Ben, unfortunately I'm not sure on a timeline, so I added a config switch for this. If you add 

```
$configArray = array(
    ...
    enable_utf8_sanitization => false,
    ...
)
```

This will disable the sanitization for you. Let me know if you have any further problems. It should be available in composer shortly as v0.18.1
